app/genssz/testdata/example.go

```diff
@@ -0,0 +1,23 @@
+// Copyright © 2022-2023 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package testdata
+
+//go:generate go install github.com/obolnetwork/charon/app/genssz
+//go:generate genssz
+
+type Foo struct {
+	ByteList []byte  `ssz:"ByteList[32]"`
+	Number   int     `ssz:"uint64"`
+	Bytes4   [4]byte `ssz:"Bytes4"`
+	Bytes2   []byte  `ssz:"Bytes2"`
```

Comment:
In case of SSZ arrays, should the field be specified as a Go array rather than a slice? (see field `Bytes4`).

If yes, maybe we could refrain from specifying the SSZ size and infer it from the Go struct field size.

Reply:
No, BytesN may also be slices.